NasaViewer/ViewControllers/APOD/MainMenuView/APODMainMenuViewController.swift

```diff
@@ -52,23 +53,50 @@ class APODMainMenuViewController: UIViewController {
             contentView.view.bottomAnchor.constraint(equalTo: view.bottomAnchor),
         ])
         self.checkforPermission()
+        self.addCancelToButton()
     }
     
     private func pushViewWithRandomArticles() {
```

Comment:
lately, it's very fashionable to use a coordinator pattern to decouple navigation from view controllers :) You should think about doing that :)